src/hooks/useGestureEventsHandlersDefault.tsx

```diff
@@ -298,7 +297,7 @@ export const useGestureEventsHandlersDefault: GestureEventsHandlersHookType =
                 absoluteY > WINDOW_HEIGHT - animatedKeyboardHeight.value
               )
             ) {
-              dismissKeyboardOnJs();
+              runOnJS(Keyboard.dismiss);
```

Comment:
This will crash with `reanimated > 3.x`. My recommendation  -> [94cf11e](https://github.com/gorhom/react-native-bottom-sheet/commit/94cf11eb43f9cc2a356da7e7967ec63baf390a74#diff-30e33da2afcfb6340f6c7eff36ac5932d6cc5e6f7f5e895d5bb4bd7e48e6d984R301)

Reply:
Hi,
I haven't tested on V3.
So it means we need to bump the required reanimated version on this lib?
Or what is your suggestion? 
That way it crashes on reanimated<3.x

Thanks

Reply:
Update:
I tested on   ` "react-native-reanimated": "~3.1.0"` and it works perfectly

Reply:
Just tested in `"react-native-reanimated": "3.1.0"`.

<div>
<img src="https://github.com/gorhom/react-native-bottom-sheet/assets/9556448/55e735a3-7bdd-4612-95b9-0b5c5bca1968)" width="220" />
</div>

Reply:
yes exactly, that is a fix that this PR has.
That was the cause of the issue.

Reply:
Tried this, but got this error now `ReanimatedError: Trying to access property `dismiss` of an object which cannot be sent to the UI runtime., js engine: reanimated`

Reply:
> Tried this, but got this error now `ReanimatedError: Trying to access property `dismiss` of an object which cannot be sent to the UI runtime., js engine: reanimated`

Same

<img src="https://github.com/gorhom/react-native-bottom-sheet/assets/61600246/713d8cdc-404a-480d-8a35-28259f755048" width="220" />
</div>

Reply:
something like this works without problem  

```tsx
const dismissKeyboard = () => {
  Keyboard.dismiss()
}

...

runOnJS(dismissKeyboard)();
```

Reply:
https://github.com/gorhom/react-native-bottom-sheet/pull/1367#issuecomment-1574224496